openapi.yaml

```diff
@@ -7773,6 +7775,9 @@ components:
           description: Name of the priority. The details are returned in `FilteredJIRAStuff.priorities`.
           nullable: true
           type: string
+        rendered_description:
+          description: The description of the issue rendered as HTML.
```

Comment:
Gaetano, where will this be used? 
Why will it be returned as HTML?

Reply:
Athena will consume  this.
Jira API will return this as HTML, load this for example:
https://athenianco.atlassian.net/rest/api/3/issue/DEV-6132?expand=renderedFields